Acrolinx.Sidebar/StartPageInstaller.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Acrolinx.Sdk.Sidebar.Util.Logging;
+using System.Diagnostics.Contracts;
+
+namespace Acrolinx.Sdk.Sidebar
+{
+    class StartPageInstaller
+    {
+        public static readonly string acrolinxStartPageDir = Path.GetTempPath() + @"acrolinx_start_page_" + GetStartPageVersion() + @"\";
+
+        internal static string GetStartPageVersion()
+        {
+            var executingAssembly = Assembly.GetExecutingAssembly();
+
+            var startPageVersion = executingAssembly.GetManifestResourceNames().
+                Where(r => r.Contains("acrolinx_sidebar_startpage") && r.Contains("version.properties")).Single();
+
+            Contract.Assume(!string.IsNullOrEmpty(startPageVersion));
+
+            using (StreamReader stremReader = new StreamReader(executingAssembly.GetManifestResourceStream(startPageVersion)))
+            {
+                startPageVersion = stremReader.ReadToEnd();
+                Contract.Assume(!string.IsNullOrEmpty(startPageVersion));
+            }
+            return startPageVersion.Split('=').Last();
+        }
+
+        internal static void ExportStartPageResources()
```

Comment:
let's check if we could read it from res:// url instead

Reply:
done.
Removed StartPageInstaller file - now we are using Startpage.dll.